src/sentry/web/frontend/debug/mail.py

```diff
@@ -470,7 +470,11 @@ def build_report(project):
         series = [
             (
                 timestamp + (i * rollup),
-                (random.randint(0, daily_maximum), random.randint(0, daily_maximum)),
+                (
```

Comment:
Can we add a comment here explaining what each value in this tuple represents?

Reply:
Like this right
```py
                (
                    # Resolved issues
                    random.randint(0, daily_maximum),
                    # Unresolved issues
                    random.randint(0, daily_maximum),
                    # Transactions
                    random.randint(0, daily_maximum),
                ),
```